<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="picture1">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="picture2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="calcpix2glo_btn">
            <property name="text">
             <string>calc pix2glo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="genert_btn">
            <property name="text">
             <string>Genert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="test_btn_2">
            <property name="text">
             <string>test</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="test_btn">
            <property name="text">
             <string>paress</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="testbtn">
            <property name="text">
             <string>testbtn</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="custom_btn">
              <property name="text">
               <string>Custom</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="autorad_btn">
              <property name="text">
               <string>Auto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="pauserad_btn">
              <property name="text">
               <string>pause</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Webcam</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cam_count"/>
          </item>
          <item>
           <widget class="QPushButton" name="chachimg_btn">
            <property name="text">
             <string>Chatch photo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Screw</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="screwtable">
          <column>
           <property name="text">
            <string>ymin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>xmin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ymax</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>xmax</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>nut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="nuttable">
          <column>
           <property name="text">
            <string>ymin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>xmin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ymax</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>xmax</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
